media-server/media_server.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#
+# Author: Fabian Solano <fabian.solano@ridgerun.com>
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+
+# Load settings
+source config.txt
+
+gstd_not_found_msg="Could not connect to localhost: Connection refused"
+check_gstd=$(gstd-client list_pipelines  2>&1)
+
+# Verify if gstd is reachable
+if [[ $check_gstd == $gstd_not_found_msg ]] ; then
+    echo "Could not connect to gstd. Please check gstd is up and running."
+    exit
+fi
+
+# Create pipelines
+echo "Initializing camera capture"
+gstd-client pipeline_create cam_pipe v4l2src device=$CAMERA_DEVICE \
+! video/x-raw, width=$CAMERA_DEVICE_WIDTH, height=$CAMERA_DEVICE_HEIGHT \
+! videoconvert ! video/x-raw, format=I420 ! videoscale \
```

Comment:
you do not need to separate the caps. You can do:  `videoscale ! videoconvert ! video/x-raw,width=640,height=480,format=I420`